lib/bundle/core+encrypt+promise.js

```diff
@@ -11,6 +11,6 @@ function Ravelin(cfg) {
   this.encrypt = new Encrypt(this.core, cfg);
 }
 
-Ravelin.Promise = window.Promise || Promise;
+Ravelin.Promise = typeof Promise !== 'undefined' && Promise || PolyfillPromise;
```

Comment:
Would it be easier to do a null check? Same goes for the other ones? 
```suggestion
Ravelin.Promise = Promise != null && Promise || PolyfillPromise;
```

Reply:
In older browsers before Promise because a thing, you'd get an equivalent of a ReferenceError because it's not a defined global.

Reply:
UGH I hate older browsers